dataproxy/service.go

```diff
@@ -140,9 +140,6 @@ func (s Service) validateCreateDownloadLocationRequest(req *service.CreateDownlo
 func createShardedStorageLocation(ctx context.Context, shardSelector ioutils.ShardSelector, store *storage.DataStore,
 	cfg config.DataProxyUploadConfig, keyParts ...string) (storage.DataReference, error) {
 	keySuffixArr := make([]string, 0, 4)
-	if len(cfg.StoragePrefix) > 0 {
```

Comment:
Since the only thing we now put in keySuffix is keyParts, I think we can get rid of the former completely and avoid the extra initialization

Reply:
Good point. I adjusted it.